docs/install-config/configure-internal-tls.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Configure Internal TLS communication between Harbor Component
+weight: 30
+---
+
+By default, The communication between Harbor's component are use http protocol which regards as insecure in production environment. From Harbor v2.0, TLS can be enabled for internal network. In production environments, always use HTTPS is a best practice.
```

Comment:
-> ' By default, The communication between Harbor's component use HTTP protocol which might not be secure enough for some production environment. Since Harbor v2.0, TLS can be enabled for internal network. In production environments, always use HTTPS is a recommended best practice.'